music/demos/common.ts

```diff
@@ -0,0 +1,203 @@
+  notes: [
+    {pitch: 69, quantizedStartStep: 0, quantizedEndStep: 2, program: 0},
+    {pitch: 71, quantizedStartStep: 2, quantizedEndStep: 4, program: 0},
+    {pitch: 73, quantizedStartStep: 4, quantizedEndStep: 6, program: 0},
+    {pitch: 74, quantizedStartStep: 6, quantizedEndStep: 8, program: 0},
+    {pitch: 76, quantizedStartStep: 8, quantizedEndStep: 10, program: 0},
+    {pitch: 81, quantizedStartStep: 12, quantizedEndStep: 16, program: 0},
+    {pitch: 78, quantizedStartStep: 16, quantizedEndStep: 20, program: 0},
+    {pitch: 81, quantizedStartStep: 20, quantizedEndStep: 24, program: 0},
+    {pitch: 76, quantizedStartStep: 24, quantizedEndStep: 32, program: 0}
+  ],
+  quantizationInfo: {stepsPerQuarter: 4}
+};
+
+export const MEL_TWINKLE: mm.INoteSequence = {
+  notes: [
+    {pitch: 60, quantizedStartStep: 0, quantizedEndStep: 2, program: 0},
+    {pitch: 60, quantizedStartStep: 2, quantizedEndStep: 4, program: 0},
+    {pitch: 67, quantizedStartStep: 4, quantizedEndStep: 6, program: 0},
+    {pitch: 67, quantizedStartStep: 6, quantizedEndStep: 8, program: 0},
+    {pitch: 69, quantizedStartStep: 8, quantizedEndStep: 10, program: 0},
+    {pitch: 69, quantizedStartStep: 10, quantizedEndStep: 12, program: 0},
+    {pitch: 67, quantizedStartStep: 12, quantizedEndStep: 16, program: 0},
+    {pitch: 65, quantizedStartStep: 16, quantizedEndStep: 18, program: 0},
+    {pitch: 65, quantizedStartStep: 18, quantizedEndStep: 20, program: 0},
+    {pitch: 64, quantizedStartStep: 20, quantizedEndStep: 22, program: 0},
+    {pitch: 64, quantizedStartStep: 22, quantizedEndStep: 24, program: 0},
+    {pitch: 62, quantizedStartStep: 24, quantizedEndStep: 26, program: 0},
+    {pitch: 62, quantizedStartStep: 26, quantizedEndStep: 28, program: 0},
+    {pitch: 60, quantizedStartStep: 28, quantizedEndStep: 32, program: 0}
+  ],
+  quantizationInfo: {stepsPerQuarter: 4}
+};
+
+export function writeTimer(elementId: string, startTime: number) {
+  document.getElementById(elementId).innerHTML =
+      ((performance.now() - startTime) / 1000).toString() + 's';
+}
+
+export function writeNoteSeqs(elementId: string, seqs: mm.INoteSequence[]) {
+  const element = document.getElementById(elementId);
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+  seqs.forEach(seq => {
+    const seqWrap = document.createElement('div');
+    const seqText = document.createElement('span');
+    seqText.innerHTML = '[' +
+        seq.notes
+            .map(n => {
+              let s = '{p:' + n.pitch + ' s:' + n.quantizedStartStep;
+              if (n.quantizedEndStep != null) {
+                s += ' e:' + n.quantizedEndStep;
+              }
+              s += '}';
+              return s;
+            })
+            .join(', ') +
+        ']';
+    seqWrap.appendChild(seqText);
+    seqWrap.appendChild(createPlayer(seq));
+    element.appendChild(seqWrap);
+  });
+}
+
+function createPlayer(seq: mm.INoteSequence) {
```

Comment:
allow you to choose between player types? we should probably keep the non-multitrack ones on the original player

Reply:
Why?

Reply:
Switched back to original player for multitrack.  Was tempted to do trios too but that's the only sequence type that uses the synth bass...

Reply:
Oops, I meant all but multitrack.